ddtrace/contrib/dbapi/__init__.py

```diff
@@ -70,18 +70,19 @@ def __enter__(self):
 class TracedConnection(wrapt.ObjectProxy):
     """ TracedConnection wraps a Connection with tracing code. """
 
-    def __init__(self, conn, pin=None):
+    def __init__(self, conn, pin=None, cursor_cls=TracedCursor):
         super(TracedConnection, self).__init__(conn)
         name = _get_vendor(conn)
         db_pin = pin or Pin(service=name, app=name, app_type=AppTypes.db)
         db_pin.onto(self)
+        self._self_cursor_cls = cursor_cls
```

Comment:
we probably don't need `self._self_cursor_cls`, just `self._cursor_cls` is probably good enough.

Reply:
Had tried that first but seems that `wrapt` requires it:

> If creating a custom proxy and it needs to keep attributes of its own which should not be saved through to the wrapped object, those attributes should be prefixed with _self_.
https://wrapt.readthedocs.io/en/latest/wrappers.html

Reply:
ah, did not know that, great find!

Reply:
maybe a comment about it?